src/status_im/core_spec.cljs

```diff
@@ -5,6 +5,5 @@
     [status-im.contexts.communities.actions.community-options.component-spec]
     [status-im.contexts.wallet.add-address-to-watch.component-spec]
     [status-im.contexts.wallet.add-address-to-watch.confirm-address.component-spec]
+    [status-im.contexts.wallet.create-account.edit-derivation-path.component-spec]
     [status-im.contexts.wallet.send.input-amount.component-spec]))
-
-  ;; [status-im.contexts.wallet.create-account.edit-derivation-path.component-spec]
```

Comment:
Btw there is one more test here (after rebasing)

Reply:
Oh thanks for catching that @OmarBasem.

Reply:
@OmarBasem, @J-Son89, when possible, please have a look at this commit I made to fix failing tests https://github.com/status-im/status-mobile/pull/18276/commits/884266bc77b913a0862d9cfae61c890b3bdcca4d. The problem appeared after I rebased, I guess from the recently merged commit 5eb0baa64dd937cbcaa8bd0eb93a56adf6498cba  

The root cause was because in `quo.components.wallet.network-bridge.view/view` we were calling `name` on a nil value, but this throws exceptions in the tests for `status-im.contexts.wallet.send.input-amount.component-spec`. I don't know if this could be a problem in production later on, but it's safer anyway to guard against calling `name` in a possibly nil value.

Reply:
LGTM @ilmotta 👍